app/models/hyrax/batch_ingest/batch.rb

```diff
@@ -22,5 +22,9 @@ def admin_set
     def collection
       # TODO: return instance of Collection object
     end
+
+    def failed_items?
+      batch_items.present? && batch_items.select { |bi| bi.error.present? }.present?
```

Comment:
There is a handy convenience method that could simplify this:
`batch_items.any?(&:error)`